readme.md

```diff
@@ -75,7 +75,7 @@ Click the type names for complete docs.
 - [`SetOptional`](source/set-optional.d.ts) - Create a type that makes the given keys optional.
 - [`SetRequired`](source/set-required.d.ts) - Create a type that makes the given keys required.
 - [`PromiseValue`](source/promise-value.d.ts) - Returns the type that is wrapped inside a `Promise` type.
-- [`OneOrMore`](source/one-or-more.d.ts) - Matches either the given `Value` type or an array of `Value`.
+- [`OneOrMore`](source/one-or-more.d.ts) - Either matches an array containing only `Value`, or `Value` itself, not in an array.
```

Comment:
I meant the other way around. As I already improved the readme description. re: https://github.com/sindresorhus/type-fest/pull/85/commits/87348b1fafd79f6b55bc8b052383d309f4ac3ab4#r379619309